### ISSUE TITLE
Placement or position

### ISSUE THREAD

**Issue by harshakn6** [fengyuanchen/datepicker#149]
This is not an issue. But can we have an option for choosing the placement of the picker?
```
{
placement: "top" //bottom
}
```

Thanks

**Comment by imkimchi** [comment]
+1, Looking for this feature

**Comment by fvonellerts** [comment]
Yeah it's not like that would be a lot of effort. Now I have to find some ugly fix for this.

**Comment by kila-sergey** [comment]
> Yeah it's not like that would be a lot of effort. Now I have to find some ugly fix for this.

How did you fix this?
